### PR TITLE
Development complete

### PR DESCRIPTION
**Pull request by gopu13**
Hello,

Development is completed.

Best Regards,
Gopu

### CODE REVIEW

File: src/zgit_incl_latest1.prog.abap [hunk 1]
```diff
@@ -0,0 +1,5 @@
+*&---------------------------------------------------------------------*
+*& Include          ZGIT_INCL_LATEST1
+*&---------------------------------------------------------------------*
+
+data ls_zgit_latest1 type ZGIT_LATEST1.
```
Comment (gopu-ibm) [inline src/zgit_incl_latest1.prog.abap:5]: Wrong naming convention

File: src/zgit_prog_latest1.prog.abap [hunk 2]
```diff
@@ -0,0 +1,14 @@
+*&---------------------------------------------------------------------*
+*& Report ZGIT_PROG_LATEST1
+*&---------------------------------------------------------------------*
+*&
+*&---------------------------------------------------------------------*
+REPORT ZGIT_PROG_LATEST1.
+
+include zgit_incl_latest1.
+
+ls_zgit_latest1 = value #( key1 = '1000' ).
```
Comment (gopu-ibm) [inline src/zgit_prog_latest1.prog.abap:10]: Naming convention issue to be checked

**Review by gopu-ibm — CHANGES_REQUESTED** [review]

File: src/zgit_incl_latest1.prog.abap (hunk 1 above)
Comment (gopu13) [reply src/zgit_incl_latest1.prog.abap:5]: Issue fixed

File: src/zgit_prog_latest1.prog.abap (hunk 2 above)
Comment (gopu13) [reply src/zgit_prog_latest1.prog.abap:10]: Issue fixed.

**Review by gopu-ibm — APPROVED** [review]

**Comment by gopu-ibm** [comment]
Review completed.

**Pull request merged**